src/Test/WebDrapo.Test/WebDrapo.Test.csproj

```diff
@@ -11,6 +11,11 @@
     <EmbeddedResource Include="Pages\*.Test.html" />
   </ItemGroup>
  
+  <ItemGroup>
```

Comment:
we dont need to change csproj to include a new test

Reply:
Was automaticaly inserted by VS when creating file by the shortcut ALT + SHIFT + A